Hunting Queries/MultipleDataSources/FirewallRuleChanges_using_netsh.yaml

```diff
@@ -0,0 +1,126 @@
+  | extend timestamp = StartTime, AccountCustomEntity = InitiatingProcessAccountName, HostCustomEntity = DeviceName
+  ),
+  (
+  Event
+  | where TimeGenerated > ago(EndTime)
+  | where Source == "Microsoft-Windows-Sysmon"
+  | where EventID == 1
+  | extend EventData = parse_xml(EventData).DataItem.EventData.Data
+  | mv-expand bagexpansion=array EventData
+  | evaluate bag_unpack(EventData)
+  | extend Key=tostring(['@Name']), Value=['#text']
+  | evaluate pivot(Key, any(Value), TimeGenerated, Source, EventLog, Computer, EventLevel, EventLevelName, EventID, UserName, RenderedDescription, MG, ManagementGroupName, Type, _ResourceId)
+  // | where OriginalFileName =~ "netsh.exe"
+  | where CommandLine has_all ("advfirewall", "firewall") and CommandLine has_any (tokens)
+  | where User !in~ (AccountAllowList)
+  | extend KeyValuePairs = extract_all(@'(?P<key>\w+)=(?P<value>[a-zA-Z0-9-\":\\\s$_@()."]+\"|[a-zA-Z0-9-\":$_\\@()."]+)', dynamic(["key","value"]), CommandLine)
+  | mv-apply KeyValuePairs on (
+    summarize CommandLineParsed = make_bag(pack(tostring(KeyValuePairs[0]), KeyValuePairs[1]))
+  )
+  | extend RuleName = tostring(parse_json(CommandLineParsed).name), Program = tostring(parse_json(CommandLineParsed).program)
+  | join kind=leftanti (
+  Event
+  | where TimeGenerated > ago(EndTime)
+  | where Source == "Microsoft-Windows-Sysmon"
+  | where EventID == 1
+  | extend EventData = parse_xml(EventData).DataItem.EventData.Data
+  | mv-expand bagexpansion=array EventData
+  | evaluate bag_unpack(EventData)
+  | extend Key=tostring(['@Name']), Value=['#text']
+  | evaluate pivot(Key, any(Value), TimeGenerated, Source, EventLog, Computer, EventLevel, EventLevelName, EventID, UserName, RenderedDescription, MG, ManagementGroupName, Type, _ResourceId)
+  // | where OriginalFileName =~ "netsh.exe"
+  | where CommandLine has_all ("advfirewall", "firewall") and CommandLine has_any (tokens)
+  | where User !in~ (AccountAllowList)
+  | extend KeyValuePairs = extract_all(@'(?P<key>\w+)=(?P<value>[a-zA-Z0-9-\":\\\s$_@()."]+\"|[a-zA-Z0-9-\":$_\\@()."]+)', dynamic(["key","value"]), CommandLine)
+  | mv-apply KeyValuePairs on (
+    summarize CommandLineParsed = make_bag(pack(tostring(KeyValuePairs[0]), KeyValuePairs[1]))
+  )
+  | extend RuleName = tostring(parse_json(CommandLineParsed).name), Program = tostring(parse_json(CommandLineParsed).program)
+  ) on RuleName, Program
+  | summarize count() , StartTime= min(TimeGenerated), EndTime=max(TimeGenerated) by Computer, User, Process, RuleName, Program, CommandLineParsed = tostring(CommandLineParsed), ParentImage
+  | extend timestamp = StartTime, AccountCustomEntity = User, HostCustomEntity = Computer
+  )
+  )
```

Comment:
We want to now do the entity mappings for Hunting queries also, this allows for future support of features.